pipeline/run_eval.py

```diff
@@ -98,6 +99,7 @@ def get_sampling_cmd(benchmark, random_seed, extra_eval_args="", extra_arguments
     wrapper_args.add_argument("--model_path", required=True)
     wrapper_args.add_argument("--server_type", choices=('nemo', 'tensorrt_llm'), default='tensorrt_llm')
     wrapper_args.add_argument("--output_dir", required=True)
+    wrapper_args.add_argument("--context_type", default="empty")
```

Comment:
You don't need to add an extra parameter here as you can always add `++prompt.context_type=table` to the end of the command and it will work. So let's remove it to not have redundant options

Reply:
(and put empty as the default back in the cmd above)

Reply:
Ah, I was originally worried that having it specified twice in the command would cause an error.

Just to make sure--will the `++prompt.context_type=table` additional arg overwrite the existing `prompt.context_type=empty` if both are specified?

Reply:
yes, it will overwrite